tests/__init__.py

```diff
@@ -0,0 +1,4 @@
+from tests.utils import Samarium
+
+
+__all__: list[str] = ["Samarium"]
```

Comment:
Unnecessary type annotation, and I'd rather use a tuple here.
```suggestion
__all__ = ("Samarium",)
```

Reply:
IDC either way cause it's not my codebase. trag1c can pick.

Reply:
> IDC either way cause it's not my codebase. trag1c can pick.

I'd prefer a tuple here